.github/workflows/msbuild.yml

```diff
@@ -50,33 +36,45 @@ jobs:
       if: matrix.os == 'ubuntu-16.04'
       run: |
         sudo apt-get install -y build-essential
-    - uses: actions/setup-python@v1
+    - name: Checkout Dafny
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        path: dafny
+    - name: Get Z3 - windows
+      if: matrix.os == 'windows-latest'
+      shell: pwsh
+      run: |
+        Invoke-WebRequest -Outfile z3X.zip -Uri https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-win.zip
+    - name: Get Z3 - mac
+      if: matrix.os == 'macos-latest'
+      run: |
+        wget https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-osx-10.14.2.zip
+    - name: Get Z3a - ubuntu
+      if: matrix.os == 'ubuntu-latest' || matrix.os == 'ubuntu-16.04'
+      run: |
+        wget https://github.com/Z3Prover/z3/releases/download/Z3-4.8.5/z3-4.8.5-x64-ubuntu-16.04.zip
+    - name: Install Z3
+      run: |
+        unzip z3*.zip && rm *.zip
+        cp -r z3* dafny/Binaries/z3
+    - name: Setup dotnet
+      uses: actions/setup-dotnet@v1.7.2
+      with:
+        dotnet-version: '3.1.x' # SDK Version to use; x will use the latest version of the 3.1 channel
+    - name: Build Dafny
+      run: dotnet build dafny/Source/Dafny.sln
     - name: Set up JDK 1.8
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - name: Build and Copy DafnyRuntime.jar with Gradle
+      run: dafny/Source/DafnyRuntime/DafnyRuntimeJava/gradlew -p dafny/Source/DafnyRuntime/DafnyRuntimeJava copyJarToBinaries
     - name: Upgrade outdated pip
       run: python -m pip install --upgrade pip
     - name: Install lit
       run: pip install lit OutputCheck pyyaml
     - uses: actions/setup-node@v1
     - run: npm install bignumber.js
-    - name: Checkout Dafny
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
-        path: dafny
-    - name: Create release
-      run: |
-        python dafny/Scripts/package.py 0.0.0-CI --os ${{ matrix.os_for_build }} --skip_manual=true --trial=true
-    - if: runner.os == 'Windows'
-      shell: pwsh
-      run: |
-        Expand-Archive dafny/Package/dafny-0.0.0-CI-x64-${{ matrix.os_for_release_zip }}.zip unzippedRelease
-    - if: runner.os != 'Windows'
-      run: |
-        unzip dafny/Package/dafny-0.0.0-CI-x64-${{ matrix.os_for_release_zip }}.zip -d unzippedRelease
-        sudo chmod +x $PWD/unzippedRelease/dafny/dafny
```

Comment:
This has completely reverted the work Remy did to test on a built release rather than on a Git working tree.
Was that intentional? If not, the removed lines here should be replaced.

Reply:
That was my mistake. I will revert it. Thanks for pointing it out.